pkgs/tools/typesetting/tex/gladtex/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "gladtex";
+  version = "unstable-2022-03-09";
```

Comment:
Only nitpick: is there a particular reason for 2022-03-09 specifically, as opposed to the latest release (seems like v3.0) or the latest tag (v3.1) or the latest master (22 Jan 2023)? If so, it would be good to have a comment about it.

LGTM otherwise, it builds and seems to work well on my machine.

Reply:
Sorry, should've documented that. The reason for why I chose master is that gladtex (surprisingly) did not work on the tags I tested. You're right though that it should be `unstable-2023-01-22` instead of `unstable-2022-03-09`, I guess I was a bit too tired when I wrote this derivation :sweat_smile:

Reply:
(PS: I don't have the power to merge this unfortunately)